src/sync/utils.rs

```diff
@@ -13,18 +13,28 @@ use datafusion_expr::{col, lit, Accumulator, Expr};
 use std::collections::{HashMap, HashSet, VecDeque};
 use tracing::log::warn;
 
+// Returns the total number of bytes and rows in the slice of batches
+pub(super) fn size_and_rows(batches: &[RecordBatch]) -> (usize, usize) {
```

Comment:
```suggestion
pub(super) fn compute_size_and_rows(batches: &[RecordBatch]) -> (usize, usize) {
```
or `get_size_and_rows`